Dockerfile

```diff
@@ -1,18 +1,15 @@
 #*********************************************************************
-# Copyright (c) Intel Corporation 2021
+# Copyright (c) Intel Corporation 2024
 # SPDX-License-Identifier: Apache-2.0
 #*********************************************************************/
 #Multistage docker layer to isolate the git credentials
 #First stage copy and install dependencies
 FROM node:20-bullseye-slim@sha256:9cb48d12eeccb9e6ad25e987dda1077399cd63877a46e9e848273c44690ca175 as builder
 LABEL license='SPDX-License-Identifier: Apache-2.0' \
```

Comment:
this label needs to be moved to below --- its labeling the build image, not the output image